benchexec/tools/legion-symcc.py

```diff
@@ -17,8 +18,10 @@ class Tool(benchexec.tools.template.BaseTool2):
     REQUIRED_PATHS = [
         "legion.sh",
         "Legion.py",
-        "__VERIFIER.c",
-        "ubuntu2004",
+        "Verifier.cpp",
+        "legion",
+        "lib",
+        "lib32",
```

Comment:
I would suggest to merge the old and new list, such that it works with both version.

Reply:
Uh, but missing required files gives an error, doesn't it? The current archive does not have the old files any more.

Reply:
No, these are just patterns for file names, and as such non-matching patterns are ignored.